modules/react-native-status/ios/RCTStatus/RCTStatus.m

```diff
@@ -536,16 +536,73 @@ - (void)handleSignal:(NSString *)signal
     callback(@[Identifier]);
 }
 
+//// deviceinfo
+
 - (bool) is24Hour
 {
     NSString *format = [NSDateFormatter dateFormatFromTemplate:@"j" options:0 locale:[NSLocale currentLocale]];
     return ([format rangeOfString:@"a"].location == NSNotFound);
 }
 
+- (NSString *)getBuildId {
+    #if TARGET_OS_TV
+        return @"not available";
+    #else
+        size_t bufferSize = 64;
+        NSMutableData *buffer = [[NSMutableData alloc] initWithLength:bufferSize];
+        int status = sysctlbyname("kern.osversion", buffer.mutableBytes, &bufferSize, NULL, 0);
```

Comment:
that might be an issue when apple reviews our code... I think for iOS the device model (iPhone8,2; etc) is enough to see if we should enable biometrics.